index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="stylesheet.css" />
+  </head>
+  <body>
+    <nav class="nav_links">
+      <img src="geo1.png" alt="cool logo" />
+      <div class="burger">
+        <div class="line1"></div>
+        <div class="line2"></div>
+        <div class="line3"></div>
+      </div>
+    </nav>
+    <section class="hero">
+      <h2>hello</h2>
+      <div class="logo">
+        Shaheer <br />
+        Chaudry
+      </div>
```

Comment:
- [ ] Kindly implement changes asked by the previous reviewer, you did not change your `<div>` tag to `<h1>` tag as requested.

Reply:
implemented but i was under the impression they were optional suggestion thus my project could be approved without their implementations